finagle-netty4/src/main/scala/com/twitter/finagle/netty4/package.scala

```diff
@@ -15,4 +15,10 @@ package object netty4 {
    */
   private[finagle] val Toggles: ToggleMap =
     StandardToggleMap("com.twitter.finagle.netty4", DefaultStatsReceiver)
+	
+  /** Stops HashedWheelTimer thread */
+  def stopHashedWheelTimer {
```

Comment:
how about we instead make the HashedWheelTimer object public and add a public method `stop` to it?  I think that will be a cleaner API.